views/lib/loggedin.hbs

```diff
@@ -247,7 +247,7 @@
                                     <li><div class="dropdown-name" data-testid="name-in-the-icon">{{ currentUser.displayName }} ({{$t (concat "global.placeholder." currentRole) }})</div></li>
                                     {{> "user/forms/language" language=@root.userLanguage }}
                                     <li><a class="dropdown-item" data-testid="settings" href="/account/" role="menuitem" aria-label="{{$t 'lib.loggedin.tab_label.settings' }}">{{$t "lib.loggedin.tab_label.settings" }}</a></li>
-                                    <li><a class="dropdown-item localstorageclear" data-testid="logout" href="/logout/" role="menuitem" aria-label="{{$t 'lib.loggedin.tab_label.signOut'}}">{{$t "lib.loggedin.tab_label.signOut"}}</a></li>
+                                    <li><a class="dropdown-item localstorageclear" data-testid="logout" href= {{#hasConfig "HYDRA_LOGOUT_URI"}} {{getConfig "HYDRA_LOGOUT_URL"}} {{else}} "/logout/" {{/hasConfig}} role="menuitem" aria-label="{{$t 'lib.loggedin.tab_label.signOut'}}">{{$t "lib.loggedin.tab_label.signOut"}}</a></li>
```

Comment:
Due to how the logout flow works it will redirect back to the schulcloud logout.
schulcloud logout is set as the post logout uri in hydra and will always be executed after all oauth2 logouts have finished.

Reply:
Okay, what about the Session at the schul cloud timed out?
Not the JWT from Hydra for nextcloud or so on.

Reply:
The autologout is so far only for bildungscloud. implementing it can be done in a followup ticket.

Reply:
okay